vignettes/bds_exposure.Rmd

```diff
@@ -520,11 +520,11 @@ adex %>%
 
```

Comment:
In the "Create Summary Records" section `derive_params_exposure` should be changed to `derive_param_exposure` (in text and code, three occurrences).

Reply:
Fixed